.github/policies/resourceManagement.yml

```diff
@@ -2539,6 +2539,16 @@ configuration:
         then:
         - addLabel:
             label: TypeSpec
+    - if:
+      - payloadType: Issue_Comment
+      then:
+      - if:
+        - commentContains:
+            pattern: "$/pr RequestMerge^"
```

Comment:
@weshaggard will this support all casings? E.g. `/PR requestmerge`

Reply:
Not sure that is something I will test once I get someone to help enable this in the private repo.